client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -54,68 +55,39 @@ class DeployedModel:
         # <DeployedModel 01234567-0123-0123-0123-012345678901>
 
     """
-    def __init__(self, _host=None, _run_id=None, _from_url=False, **kwargs):
-        # this is to temporarily maintain compatibility with anyone passing in `socket` and `model_id` as kwargs
-        # TODO: instate `host` and `run_id` params
-        # TODO: remove the following block of param checks
+    def __init__(self, prediction_url, token=None, creds=None):
         # TODO: put automodule verta.deployment back on ReadTheDocs
```

Comment:
If we could remove this line as well, since `verta.deployment` is being rendered in our docs now 🙇🏼 
```suggestion
```